Aufgabe1/gta_v1/public/index.html

```diff
@@ -39,6 +39,12 @@ <h2 class="discovery__title">Discovery</h2>
 
             <!-- ############### VS1Lab 1a Form 2 Start -->
             <form class="discovery__search" id="discoveryFilterForm" action="/discovery" method="post">
+                <fieldset>
+                    <legend>Search</legend>
+                        <label id="discovery_search_label" for="discovery_search_input">Search term or keywords:</label><br>
```

Comment:
We are not using `<br>`-Tags to break two elements. We design it in style.css file.

```suggestion
                        <label id="discovery_search_label" for="discovery_search_input">Search term or keywords:</label>
```